about.html

```diff
@@ -0,0 +1,140 @@
+          <li><a href="#player-section">Players</a></li>
+          <li><a href="#partner-section">Partner</a></li>
+          <li class="nav-btn"><a href="#events-section">Events</a></li>
+        </ul>
+      </nav>
+    </section>
+  </header>
+  <main>
+    <article class="headline-container">
+      <section class="headline about-headline">
+        <h1 id="main-heading" class="text-center ms-md-5 me-md-5 ps-md-5 pe-md-5">
+          <span class="greeting text-center"> "Hello! Snooker World" </span>
+          World Snooker Championship 2022
+        </h1>
+        <p class="p-4 bg-white">
+          You’ve probably heard the term ‘Race to the Crucible’ mentioned at some points throughout any given season.
+          This refers specifically to the race to be in the Top 16 as these individuals in the rankings automatically
+          qualify for the Crucible and avoid underdoing the stress of qualifying. Once you hear what the qualifiers have
+          to go through just to get to the main stage, you’ll understand why being in and around the Top 16 of the world
+          rankings is a desirable prospect for all professionals. It’s also why the tournament before the WSC is such an
+          important event for those players. The Top 16 are then seeded, with the defending champion holding the No. 1 seed, and the rest based off their
+          world rankings. None of these players will contest in the first round. The Top 16 face off against the 16
+          individuals that came through the qualifying rounds. As for those 16 that survived the enduring stages of
+          qualifying, this is what they have to go through.
+        </p>
+        <p class="ps-5 pe-5 text-center">
+          Please contact us per Email for any further questions about World Snooker Championship 2022 <br />
+          <a href="#" class="text-center">worldsnooker2022@gmail.com</a>
+        </p>
+      </section>
+    </article>
+    <article class="about-logo d-flex flex-column justify-content-center align-items-center p-4 border-bottom">
+      <h2 class="section-heading p-3 text-center mb-0">
+        World Snooker Championship 2022 Logo
+      </h2>
+      <hr class="mt-0" />
+      <p class="text-center">
+        This logo was decided through the logo competition from 8.June to
+        7.July.
+      </p>
+      <section class="border m-2 mt-5 p-4 d-flex align-items-center shadow">
+        <div class="display-1 me-1 rounded-circle bg-warning p-3">
+          <img width="60px" height="80px" src="img/main-icon.png" alt="icon">
+        </div>
+        <div>
+          <h2 class="h2 font-weight-bold">
+            World Snooker <br />Championship 2022
+          </h2>
+          <p class="border-top border-bottom text-center text-muted">
+            16th April - 02nd May 2022 | Sheffield, England
+          </p>
+        </div>
+      </section>
+    </article>
+
+    <article class="event-gallery p-3 text-center d-flex flex-column">
```

Comment:
- [ ] here again the right semantic tag is `section` tag as these are sections of the page